source/templates/handlebars-basics.md

```diff
@@ -52,6 +52,44 @@ controllers and components.
 
 ### Helpers
 
-Helpers bring a minimum of logic into Ember templating. Ember ships with several
-built-in helpers, which are explained in the following guides, and also allows
-you to [write your own helpers](../writing-helpers/).
+Ember gives you the ability to [write your own helpers](../writing-helpers/), to bring a minimum of logic into Ember templating.
+
+For example, let's say you would like the ability to add a few numbers together, without needing to define a computed property everywhere you would like to do so.
+
+```javascript
+// helpers/sum.js
+export default Ember.Helper.helper(function(params) {
```

Comment:
This snippet helper does not conform to the ones generated by CLI, can you tweak it please?